routes/goldap.js

```diff
@@ -265,19 +265,25 @@ module.exports = {
 
     add: async function(user, fid) {
         let group = null;
-        try {
-            group = await utils.mongo_groups().findOne({'name': user.group});
-        } catch(e) {
-            logger.error(e);
-            throw e;
+        if (!CONFIG.general.disable_user_group) {
+            try {
+                group = await utils.mongo_groups().findOne({'name': user.group});
+            } catch(e) {
+                logger.error(e);
+                throw e;
+            }
         }
-
         try {
             let created_file = await filer.ldap_add_user(user, group, fid);
             logger.debug('File created', created_file);
-            let group_dn = await get_group_dn(group.name);
-            created_file = await filer.ldap_add_user_to_group(user,group_dn, fid);
-            logger.debug('File created', created_file);
+
+            // should we do if (group) ?
+            if (!CONFIG.general.disable_user_group) {
```

Comment:
Can't we just test group not null to avoid config many times.

Reply:
https://github.com/genouest/genouestaccountmanager/pull/144/commits/74c65173ac96d3cbfafdce8477501dfb930a403a